coderd/database/dbfake/dbfake.go

```diff
@@ -5642,6 +5642,26 @@ func (q *FakeQuerier) UpdateTemplateACLByID(_ context.Context, arg database.Upda
 	return sql.ErrNoRows
 }
 
+func (q *FakeQuerier) UpdateTemplateAccessControlByID(_ context.Context, arg database.UpdateTemplateAccessControlByIDParams) error {
+	if err := validateDatabaseType(arg); err != nil {
+		return err
+	}
+
+	q.mutex.Lock()
+	defer q.mutex.Unlock()
+
+	for idx, tpl := range q.templates {
+		if tpl.ID != arg.ID {
+			continue
+		}
+		tpl.RequireActiveVersion = arg.RequireActiveVersion
+		q.templates[idx] = tpl
```

Comment:
these lines can be simplified


```suggestion
		q.templates[idx].RequireActiveVersion = arg.RequireActiveVersion
```